src/app.js

```diff
@@ -1 +1,29 @@
 'use strict';
+
+const params = process.argv.slice(2);
+const operation = params[0];
+const numbers = params.slice(1).map(Number).filter(number => !isNaN(number));
+let result;
+
+switch (operation) {
+  case 'add':
+    if (numbers.length === 0) {
+      result = 'There is nothing to add';
+    } else {
+      result = numbers.reduce((a, b) => a + b, 0);
+    }
```

Comment:
ternaries usually look better than `if/else`
```suggestion
    result = numbers.length < 1
      ? 'There is nothing to add'
      : numbers.reduce((a, b) => a + b, 0);
```